natgeo/mortar
# [SYTO-885] Fixes bug where first numeral was not changing color on card hover

To Test

- Pull SYTO-885-2
- Run `npm test` to verify tests/linting
- Run `gulp serve`
- Navigate over to http://localhost:3000/contentPackages/ and verify the #1 changes from yellow to white on hover
